### DISCUSSION TITLE
NPTEL EXAM

### DISCUSSION THREAD

**Discussion by tony1970** [community/community#32703] [GitHub Education]
Is anyone is having proctored exam (2018/2019/2020/2021/2022) question paper of EMBEDDED SYSTEM DESIGN WITH ARM of NPTEL or PYTHON FOR DATA SCIENCE please post the questions or give the link?

**Comment by samus-aran** [comment]
Hi @tony1970 ,

Thanks for being a part of the GitHub Community! We're glad you're here.

We ask that members contribute in a positive and constructive way. If you're looking for help for a particular exam, you might want to try asking for help elsewhere. GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Asking for exam questions or answers is discouraged on the forums.

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
